src/controllers/hello.controller.ts

```diff
@@ -0,0 +1,12 @@
+import type { Request, Response } from 'express';
+
+const getHello = (
+  req: Request,
+  res: Response,
+): void => {
+  res.send('hello!');
```

Comment:
I know the point here is setting up infrastructure, but there is a very close analogue to this that I think would be immediately useful: [API health](https://www.ibm.com/garage/method/practices/manage/health-check-apis)!

A first draft of that would be of equal complexity, as we don't yet have any downstream dependencies to check, and so the answer would always be healthy. Additionally, as we want this to be a JSON API (right? maybe I just assumed that and we never actually discussed it), that'd prompt us to include the JSON wiring as part of this, rather than returning a bare string.

Finally, a health check API would persist, rather than need to be torn down once we introduce the more relevant endpoints later.

This is a bigger change request than the others, and I'm afraid I don't have enough of the necessary context in my head to make a more concrete suggestion. Happy to work on this and/or pair with you, though, if you want!

Reply:
A fine suggestion!  I was a little 😬 about including throwaway in the PR just to signal structure, so this will be perfect.